src/css/_MobileNav.scss

```diff
@@ -142,3 +142,26 @@ html.mm-background .BookReader {
 @media (min-width: ($brBreakPointMobile + 1)) {
   .BRtoolbar.responsive { display: block; }
 }
+
+//strange css to get two lines ellises https://stackoverflow.com/questions/5269713/css-ellipsis-on-second-line
+li.table-contents-el{
+  overflow: hidden;
+  text-overflow: ellipsis;
+  -webkit-line-clamp: 2;
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  color: gray;
+  font-weight: lighter;
```

Comment:
Let's not change the font-weight; `font-weight: lighter` is hard to read on normal sized fonts.

Reply:
done